### ISSUE TITLE
Learning rate

### ISSUE THREAD

**Issue by seann999** [OpenHelix-Team/VLA-Adapter#43]
The paper states 1e-4, but the train config default is 5e-4. Is there any reason for this discrepancy?
